examples/deploy/addon/helloworld-helm/resources/cluster_role.yaml

```diff
@@ -38,7 +38,7 @@
       verbs: ["update", "patch"]
     - apiGroups: ["addon.open-cluster-management.io"]
       resources: ["clustermanagementaddons"]
-      verbs: ["get", "list", "watch"]
+      verbs: ["get", "list", "watch", "patch"]
```

Comment:
note, we will need users to update the clusterrole when upgrading the library.

Reply:
should be reflected in the release notes.